pkg/config/config.go

```diff
@@ -99,6 +99,13 @@ func init() {
 	Datadog.SetDefault("expvar_port", "5000")
 	Datadog.SetDefault("auth_token_file_path", "")
 
+	// Exponential backoff and circuit breaker settings
```

Comment:
I think we should prefix them with `forwarder_` like we did for the other config option.
Also Could you add some documentation somewhere on how to use those variable. I think the readme of the forwarder would be a good place. I think we should explain what those values do and why we have min/max limit for them.